### PR TITLE
fix file upload box with long filename - fix mobile sizing for 'vehicle' card on transport supp app

### PR DESCRIPTION
**Pull request by jenbeckett**

### CODE REVIEW

File: frontend/src/components/supp-allowances/TransportationAllowance.vue [hunk 1]
```diff
@@ -40,10 +40,10 @@
   <div v-for="(model, index) in models" :key="model.supplementaryApplicationId ? model.supplementaryApplicationId : model.id" @input="update(model)">
     <v-card class="my-10" :class="{ greyTop: readOnly(model), 'basic-card': !readOnly(model) }" :disabled="readOnly(model)">
       <v-row class="pa-7 pt-10">
-        <v-col cols="1">
+        <v-col cols="10" md="1">
```
Comment (vietle-cgi) [inline frontend/src/components/supp-allowances/TransportationAllowance.vue:43]: I think md="2" lg="1" would look better

image

File: frontend/src/components/ui/AppDocumentUpload.vue [hunk 2]
```diff
@@ -170,4 +170,9 @@ export default {
 .addFileButton {
   font-size: 16px;
 }
+
+:deep(.v-field__input) {
```
Comment (vietle-cgi) [inline frontend/src/components/ui/AppDocumentUpload.vue:174]: what is this change for? I tried uploading a file with long name but I don't see any difference.
image

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/supp-allowances/TransportationAllowance.vue (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/components/supp-allowances/TransportationAllowance.vue:43]: done

Comment (vietle-cgi) [reply frontend/src/components/supp-allowances/TransportationAllowance.vue:43]: I'll approve this PR once you push your new commit

**Review by vietle-cgi — APPROVED** [review]

**Pull request merged**